src/components/settings/SortableItem.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useSortable } from "@dnd-kit/sortable";
+import { CSS } from "@dnd-kit/utilities";
+import { Item, Card, Label, Icon } from "semantic-ui-react";
+
+export function SortableItem(props) {
+  const { attributes, listeners, setNodeRef, transform, transition } =
+    useSortable({ id: props.id });
+
+  const style = {
+    transform: CSS.Transform.toString(transform),
+    marginBottom: 5,
+    transition,
+  };
+
+  return (
+    <div ref={setNodeRef} style={style} {...attributes} {...listeners}>
+      <Card style={{ width: 325 }}>
+        <Card.Content>
+          <Card.Header>{props.item.name}</Card.Header>
+          <Card.Meta>Rule Type: {props.item.rule_type}</Card.Meta>
+          <Card.Description>
+            {props.item.exif_tag ? (
+              <li>Use Exiftag {props.item.exif_tag}</li>
```

Comment:
There are a bit more options that could be encountered here. I think that instead we could go through all elements of `props.item` and explicitly exclude `transform_tz/source_tz/report_tz`, then instead of hardcoded `Use file property `, `Use Exiftag ` etc. we could do something like `settingName="datetime_rule_param_" + param_name` and use either `t(settingName)` or just `settingName` with a space if there is no translation setting for `settingName`. This way we can get automatic "support" for showing any parameters that might be encountered even if frontend doesn't have specific rules for them.

`transform_tz/source_tz/report_tz` - or any other set of parameters could be explicitly excluded from this to be added separately same way how you currently do it.

What do you think?

Reply:
I implemented that unknown rules get formatted nicely, that we can exclude a couple of attributes and also added the translation string for each rule 👍